packages/api/src/routes/sensorRoute.js

```diff
@@ -34,6 +34,7 @@ router.post(
 router.delete('/delete_sensor/:sensor_id', SensorController.deleteSensor);
 router.post('/add_reading/:partner_id', SensorController.addReading);
 router.post('/get_readings', SensorController.getAllReadingsBySensorId);
+router.get('/sensor_readings/:farm_id/:days?', SensorController.getReadingsByFarmId);
```

Comment:
Have you tested this without a days value in the URL? I suspect you mean the ? to indicate that days is optional, but that's not how it usually works with Express. From the documentation, it's not clear how ? would combine with a route param like `:days`

Reply:
When I tested with Postman, it worked with days as optional parameter. I added it just in case some other functions need to use the endpoint without a fixed amount of readings.

Reply:
I was worried about that because I could not find it documented as meaning an optional parameter. The closest I can find is quoted below. Could you please test if `(?)` works and use that if it does? I'd feel safer.

> To have more control over the exact string that can be matched by a route parameter, you can append a regular expression in parentheses (()):
> 
> `Route path: /user/:userId(\d+)`
> `Request URL: http://localhost:3000/user/42`
> `req.params: {"userId": "42"}`

Reply:
Also, if you haven't, test with multidigit values for days. The question mark may only match one character.

Reply:
The endpoint seems to work with multidigit values for days as well. We could remove `?` since currently the endpoint isn't used by other features to be safe.